modules/oci/services/grafana.scm

```diff
@@ -148,6 +131,23 @@ will read the SMTP password."
    (string "alert@example.org")
    "The sender of the email alerts Grafana will send."))
 
+(define (gf-serialize-grafana-smtp-configuration field-name value)
+  (define password-file (grafana-smtp-configuration-password-file value))
+  (define password (grafana-smtp-configuration-password value))
+  #~(string-append
+     "[smtp]\n"
+     #$(serialize-configuration
+        value grafana-smtp-configuration-fields)
+     #$(if (maybe-value-set? password-file)
+           (string-append "password = $__file{"
+                          ((serialize-sops-secret
+                            'password-file password-file)
+                           %grafana-secrets-directory)
+                          "}\n")
+           (if (string-null? password)
+               ""
+               (string-append "password = " password "\n")))))
+
```

Comment:
It doesn't seem this procedure changed, unless I'm misreading. I would leave it in its original place if there is no reason to move it here, if there is a reason that I'm missing let me know!